employee-info-mgmt-system/src/main/java/Main.java

```diff
@@ -4,17 +4,18 @@
 
 import java.util.List;
 import java.util.Scanner;
+import java.util.logging.Logger;
 
 public class Main {
-
+    private static final Logger logger = Logger.getLogger(Main.class.getName());
```

Comment:
for final static identifiers we use all caps. examples: LOGGER, SOME_VARIABLE, etc. please change accordingly